src/app/(proper_react)/redesign/(authenticated)/user/welcome/View.tsx

```diff
@@ -41,6 +42,25 @@ export const View = ({
   const skipInitialStep = stepId === "enterInfo";
   const [currentStep, setCurrentStep] = useState<StepId>(stepId);
   const router = useRouter();
+  const record = useTelemetry();
+
+  useEffect(() => {
+    let pageName = "welcome";
+    if (currentStep === "enterInfo") {
+      pageName = "enter_scan_info";
+    } else if (currentStep === "findExposures") {
+      pageName = "scanning_for_exposures";
```

Comment:
Wouldn't this be the `utm_source`, i.e. the site/page that sent the user here? https://en.wikipedia.org/wiki/UTM_parameters#UTM_parameters

Reply:
Yeah, I was going back and forth on that one. I believe you are right — updated in [d1cb12e](https://github.com/mozilla/blurts-server/pull/3823/commits/d1cb12ea32885520c169ac60edc2ec4c035639c4).